examples/official/trial/cifar10_cnn_pytorch/model_def.py

```diff
@@ -68,41 +66,42 @@ def build_model(self) -> nn.Module:
             nn.ReLU(),
             nn.Dropout2d(self.context.get_hparam("layer3_dropout")),
             nn.Linear(512, NUM_CLASSES),
-        )
+        ))
 
         # If loading backbone weights, do not call reset_parameters() or
         # call before loading the backbone weights.
-        reset_parameters(model)
-        return model
+        reset_parameters(self.model)
```

Comment:
can we call `self.model.reset_parameters()` instead?

Reply:
same.